packages/experiments/src/index.js

```diff
@@ -0,0 +1,59 @@
+const CORE_MODULES_USING_EXPERIMENTS = [
+	'@wordpress/data',
+	'@wordpress/block-editor',
+	'@wordpress/block-library',
+	'@wordpress/blocks',
+	'@wordpress/core-data',
+	'@wordpress/date',
+	'@wordpress/edit-site',
+	'@wordpress/edit-widgets',
+];
+
+const accessTokens = [];
+const registeredExperiments = {};
+
+export function registerAccessToken( accessToken, packageName ) {
+	if ( ! CORE_MODULES_USING_EXPERIMENTS.includes( packageName ) ) {
+		throw new Error(
+			`Cannot register non-gutenberg package ${ packageName }.`
+		);
+	}
+	if ( packageName in registeredExperiments ) {
+		throw new Error( `Package ${ packageName } is already registered.` );
+	}
+	if (
+		! accessToken.i_realize_my_code_will_break_in_a_few_months_once_the_experimental_apis_are_removed
```

Comment:
Is this long key necessary here? As far as I understand, we need to store some unique references for every package that exposes or consumes experiments. As for that, I believe we would pass an empty object `{}` that is probably the simplest way to create a unique reference. The validation would be as simple as an iteration through all registered tokens using `===` comparison.

Reply:
It's not necessary, it's just a deterrent for anyone who tries to work around that.

Reply:
I see 😅

Reply:
I updated that part to require a string as access tokens don't make sense anymore.